app/src/main/java/helium314/keyboard/latin/suggestions/SuggestionStripView.java

```diff
@@ -375,10 +373,13 @@ public boolean onLongClick(final View view) {
     }
 
     private void onLongClickToolKey(final View view) {
-        if (ToolbarKey.CLIPBOARD == view.getTag() && view.getParent() == mPinnedKeys) {
-            onLongClickClipboardKey(); // long click pinned clipboard key
+        final ToolbarKey tag = (ToolbarKey) view.getTag();
```

Comment:
Better make sure this line does not crash, e.g. you can check `instanceof` before the cast.
(I know that currently `onLongClickToolKey` is not called if the tag is not a toolbar key, but better be safe)

Reply:
will do